packages/leafygreen-provider/src/UsingKeyboardContext.spec.tsx

```diff
@@ -59,13 +60,21 @@ describe('packages/leafygreen-provider/UsingKeyboardProvider', () => {
     });
   });
 
-  test(`usingKeyboard is false after mousedown event fires`, () => {
+  test(`usingKeyboard is true after userEvent.tab`, () => {
```

Comment:
Adding these tests since I initially thought I needed `usingKeyboard` and was having some issues. No longer need it, but leaving these tests doesn't hurt